Services/Object/classes/class.ilObject.php

```diff
@@ -341,9 +341,9 @@ public function read()
 	* @access	public
 	* @return	integer	object id
 	*/
-	function getId()
+	function getId() : int
```

Comment:
Hi @iszmais,

I'm certainly a supporter of strong type hinting in a project like this, but I'm afraid that a step like this needs a little more preparation than a PR. `ilObject` is a central class, breaking changes here should be communicated loud, clear and with enough lead time.

Best regards!